run_faster_whisper.py

```diff
@@ -0,0 +1,55 @@
+from faster_whisper import WhisperModel
+from segment import segment_audio, segment_video
+from embed import load_model, embed_audio
+#from embed import embed_audio
+import tensorflow as tf
+import torchaudio, torch
+import os
+from WavLM import WavLM, WavLMConfig
+
+#REMINDER: using the .local faster_whisper
+
+model_size = "large-v3"
```

Comment:
Speaking from experience, v3 performs slightly worse than v2 (in all cases that I evaluated on for my project it was worse).